### PR TITLE
Support a build script for Tizen target

### LINKED ISSUE #847: A shell script to build Tizen target
I  can find appropriate build scripts for Windows, iOS, Android, Raspbian, and Tegra at the below webpage.
* https://github.com/caffe2/caffe2/tree/master/scripts

Where can I find a build script in case that I want to build the Tizen target (https://www.tizen.org)?

### PR DESCRIPTION
**Pull request by leemgs**
There does not exist appropriate build script for Tizen software platform.
This commit is to fix #847.

Signed-off-by: Geunsik Lim <geunsik.lim@samsung.com>

### CODE REVIEW

File: scripts/build_tizen.sh [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+# 
+# This script shows how one can build a Caffe2 binary for raspbian. The build
+# is essentially much similar to a host build, with one additional change
+# which is to specify -mfpu=neon for optimized speed.
+
+
+# Download caffe2 source code
+# If you do not have a Tizen device, Please, run qemu-arm based chroot command. 
+# $ sudo chroot ~/tizen-rootfs qemu-arm-static /usr/bin/bash 
+git clone --recursive https://github.com/caffe2/caffe2.git && cd caffe2
```
Comment (Yangqing) [inline scripts/build_tizen.sh]: You wouldn't need this - if you have this script you should already have checked out the repository.

Comment (leemgs) [reply scripts/build_tizen.sh]: Thanks. updated with squash.

**Comment by leemgs** [comment]
\CC: @myungjoo , @jijoongmoon , @wonsubkim

**Comment by leemgs** [comment]
* After this PR: **Build is Okay.!!!!** 
 
```
tizen4.0# cd caffe2.git
tizen4.0# git checkout v0.7.0   (commit no.  f71695d)
tizen4.0# ./scripts/build_tizen.sh
     . . . Upper Omission . . . 
[100%] Built target text_file_reader_utils_test
Linking CXX executable binaries/typeid_test
[100%] Built target typeid_test
Linking CXX executable binaries/utility_ops_test
[100%] Built target utility_ops_test
Linking CXX executable binaries/workspace_test
[100%] Built target workspace_test

real	41m4.901s
user	148m25.972s
sys	0m43.360s
bash-3.2# 
```


*  Hardware specification for Caffe2 build: CPU, Memory, HDD, and Graphic.
```
invain@u1604:~$ sudo dmidecode --type=processor | grep CPU
	Version: Intel(R) Core(TM) i5-3570 CPU @ 3.40GHz

invain@u1604:~$ free
              total        used        free      shared  buff/cache   available
Mem:       16306332     6165812      510236      113348     9630284     9617236
Swap:       4194300        9436     4184864

invain@u1604:~$ sudo smartctl -i /dev/sda | grep "Device Model"
Device Model:     Samsung SSD 850 EVO 500GB

invain@u1604:~$ lspci | grep VGA
00:02.0 VGA compatible controller: Intel Corporation Xeon E3-1200 v2/3rd Gen Core processor Graphics Controller (rev 09)
```
 * Parallel build screenshot using QEMU (V-Type1)
caffe2-build-top

**Comment by leemgs** [comment]
> continuous-integration/travis-ci/pr  — The Travis CI build failed 

@Yangqing , I have completed a self-verification after applying your review. From my experiment, the build test is okay as I summarized above. Could you give me a comment  or person in charge on "The Travis Ci build failed"?

**Comment by Yangqing** [comment]
This looks great - since it is standalone file let's go ahead and merge it. The travis build failure is a separate issue.

**Pull request closed without merging**